pydeseq2/dds.py

```diff
@@ -866,6 +867,8 @@ def _refit_without_outliers(
         # Only replace if genes are not all zeroes after outlier replacement
         to_replace[to_replace] = ~new_all_zeroes
 
+        print(to_replace)
```

Comment:
remove?

Reply:
Indeed, you caught me red-handed doing print debugging :)